components/x-podcast-launchers/src/PodcastLaunchers.css

```diff
@@ -24,3 +24,22 @@
 .rss-url__copy-button {
 	padding-left: 0px;
 }
+
+.rss-url__copy-span {
+	-webkit-user-select: text;
```

Comment:
Should autoprefixer take care of this for us? n-ui should run styles through it, and I'm pretty sure it will be fine in the app.

Reply:
@benbarnett Thank you for pointing this. I have checked next-article and ft-app provide those vendor prefixes.